tests/geo/test_find_in_string.py

```diff
@@ -0,0 +1,481 @@
+import unittest
+
+from text_scrubber.geo.find_in_string import (
+    find_city_in_string,
+    find_country_in_string,
+    find_region_in_string,
+    Match
+)
+
+
+class FindCountryInStringTest(unittest.TestCase):
+    def test_find_single_country_in_string(self):
+        """
+        When the input string is distorted too much it should return an empty list
```

Comment:
This comment is misplaced. Please check all your docstrings and correct them where needed